### PR TITLE
Add log-manager service

### PR DESCRIPTION
**Pull request by wangdian**

### CODE REVIEW

File: src/log-manager/build/log-manager-logrotate.dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+FROM blacklabelops/logrotate:1.2
```
Comment (abuccts) [inline src/log-manager/build/log-manager-logrotate.dockerfile]: seems this file is unnecessary, you can use `blacklabelops/logrotate:1.2` in yaml directly.

File: src/log-manager/config/log-manager.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Microsoft Corporation
+# # All rights reserved.
+# #
```
Comment (abuccts) [inline src/log-manager/config/log-manager.yaml]: duplicate `#`

File: src/log-manager/deploy/log-manager.yaml.template [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+      annotations:
+        port: "{{ cluster_cfg["log-manager"]["port"] }}"
+    spec:
+      hostNetwork: false
+      containers:
+      - name: log-manager-logrotate
+        image: {{ cluster_cfg["cluster"]["docker-registry"]["prefix"] }}log-manager-logrotate:{{ cluster_cfg["cluster"]["docker-registry"]["tag"] }}
+        imagePullPolicy: Always
+        securityContext:
+          privileged: true
```
Comment (abuccts) [inline src/log-manager/deploy/log-manager.yaml.template]: is this needed?

File: src/log-manager/src/conf/nginx.conf [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    location = /healthz {
+      default_type text/plain;
+      return 200 "Log manager ready.";
+    }
+
+    #
+    # Get log for job
+    #
+    location /logs {
+      alias /usr/local/pai/logs;
```
Comment (abuccts) [inline src/log-manager/src/conf/nginx.conf]: add `default_type text/plain`?

it seems alias does not work

File: src/log-manager/build/log-manager-logrotate.dockerfile (hunk 1 above)
Comment (wangdian) [reply src/log-manager/build/log-manager-logrotate.dockerfile]: I suggest to leave it here for further customization.

File: src/log-manager/config/log-manager.yaml (hunk 2 above)
Comment (wangdian) [reply src/log-manager/config/log-manager.yaml]: Will remove it

File: src/log-manager/deploy/log-manager.yaml.template (hunk 3 above)
Comment (wangdian) [reply src/log-manager/deploy/log-manager.yaml.template]: No, will remove

File: src/log-manager/src/conf/nginx.conf (hunk 4 above)
Comment (wangdian) [reply src/log-manager/src/conf/nginx.conf]: It works, just not deployed on cluster

File: src/log-manager/build/log-manager-logrotate.dockerfile [hunk 5]
```diff
@@ -1,3 +1,4 @@
+# Copyright (c) 2015 Steffen Bleul
```
Comment (abuccts) [inline src/log-manager/build/log-manager-logrotate.dockerfile]: Is there any license issue for this file? You should also remove ms license.

**Pull request merged**